pkg/meta/meta.go

```diff
@@ -1486,3 +1488,49 @@ func (m *Meta) SetSchemaDiff(diff *model.SchemaDiff) error {
 	metrics.MetaHistogram.WithLabelValues(metrics.SetSchemaDiff, metrics.RetLabel(err)).Observe(time.Since(startTime).Seconds())
 	return errors.Trace(err)
 }
+
+// GroupRUStats keeps the ru consumption statistics data.
+type GroupRUStats struct {
+	ID            int64             `json:"id"`
+	Name          string            `json:"name"`
+	RUConsumption *rmpb.Consumption `json:"ru_consumption"`
+}
+
+// DailyRUStats keeps all the ru consumption statistics data.
+type DailyRUStats struct {
+	EndTime time.Time      `json:"date"`
+	Stats   []GroupRUStats `json:"stats"`
+}
+
+// RUStats keeps the lastest and second lastest DailyRUStats data.
+type RUStats struct {
+	Latest   *DailyRUStats `json:"latest"`
+	Previous *DailyRUStats `json:"previous"`
+}
+
+// GetRUStats load the persisted RUStats data.
+func (m *Meta) GetRUStats() (*RUStats, error) {
+	data, err := m.txn.Get(mRequestUnitStats)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	var ruStats *RUStats
+	if data != nil {
+		ruStats = &RUStats{}
+		if err = json.Unmarshal(data, &ruStats); err != nil {
+			return nil, errors.Trace(err)
+		}
+	}
+	return ruStats, nil
+}
+
+// SetRUStats persist new ru stats data to meta storage.
+func (m *Meta) SetRUStats(stats *RUStats) error {
+	data, err := json.Marshal(stats)
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	err = m.txn.Set(mRequestUnitStats, data)
```

Comment:
I wonder if this kv will be very large with thousands of groups and many mvcc versions by running long time?

Reply:
The stats data for a single group is smaller than 1kb, so the stats data should be smaller than 1mb for 1k resource groups, this should be acceptable. Outdated mvcc version should be GCed after new kv is written.